nginx.tmpl

```diff
@@ -0,0 +1,177 @@
+worker_processes	1;
+
+events {
+    worker_connections	1024;
+}
+
+{{$SSL := or (key "config/mantlui/ssl") (or (env "MANTLUI_SSL") "false") | parseBool}}
+{{$AUTH := or (key "config/mantl/auth") (or (env "MANTLUI_AUTH") "false") | parseBool}}
+{{$consulSSL := or (key "config/mantl/consul_ssl") (or (env "CONSUL_SSL") "false") | parseBool}}
+
+http {
+    include		mime.types;
+    default_type	application/octet-stream;
+
+    sendfile		on;
+    keepalive_timeout	65;
+
+    {{$mesos := service "leader.mesos"}}{{with $mesos}}
+    upstream mesos { {{range $mesos}}
+        server {{.Address}}:{{.Port}};{{end}}
+    } {{end}}
+
+    {{$marathon := service "marathon"}}{{with $marathon}}
+    upstream marathon { {{range $marathon}}
+        server {{.Address}}:{{.Port}};{{end}}
+    } {{end}}
+
+    {{$chronos := service "chronos"}}{{with $chronos}}
+    upstream chronos { {{range $chronos}}
+        server {{.Address}}:{{.Port}};{{end}}
+    } {{end}}
+
+    {{$consul := service "consul"}}{{with $consul}}
+    upstream consul { {{range $consul}}
+        server {{.Address}}:8500;{{end}}
+    } {{end}}
+
+    {{$mantlapi := service "mantl-api"}}{{with $mantlapi}}
+    upstream mantlapi { {{range $mantlapi}}
+        server {{.Address}}:{{.Port}};{{end}}
+    } {{end}}
+
+    {{$traefikAdmin := service "traefik-admin"}}{{with $traefikAdmin}}
+    upstream traefik-admin { {{range $traefikAdmin}}
+        server {{.Address}}:{{.Port}};{{end}}
+    } {{end}}
+
+    {{if $SSL}}
+    server {
+        listen 80;
+        return 301 https://$host$request_uri;
+    }
+    {{end}}
+
+    server {
+        listen {{if $SSL}}443 ssl{{else}}80{{end}};
+
+        {{if $SSL}}
+        ssl_certificate     {{or (env "CERTFILE") "/etc/mantlui/mantlui.cert"}};
```

Comment:
@BrianHicks did you have any plans to do something additional with SSL certs or is the intention to add CERTFILE and KEYFILE with the correct file paths to the mantlui service's environment?